clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -118,9 +134,8 @@ object IoUtil extends IoUtil {
       runGsUtilAndGetExitCode(Seq("-q", "stat", path))
     }
 
-    def md5Hash(path: String): String = {
-      val output = runGsUtilAndGetStdout(Seq("hash", "-m", "-h", path))
-      output.lines.filter(_.contains("md5")).next.split(":")(1).trim
+    def hash(path: String): String = {
+      runGsUtilAndGetStdout(Seq("hash", "-m", "-h", path))
     }
 
     def runGsUtilAndGetStdout(gsUtilArgs: Seq[String]): String = {
```

Comment:
I can't comment on line 151 directly; this code should delete the directory immediately after the gsutil command completes instead of waiting for shutdown to do it.